implementations/rust/ockam/ockam/src/protocols/stream/requests.rs

```diff
@@ -10,7 +10,8 @@ pub struct CreateStreamRequest {
 }
 
 impl CreateStreamRequest {
-    #[allow(clippy::new_ret_no_self)]
+    //noinspection ALL
+    #[allow(dead_code, clippy::new_ret_no_self)]
```

Comment:
I guess this function is not exposed from the library. Should we fix that?

Reply:
Yes, will expose